doc/conf.py

```diff
@@ -144,7 +144,6 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 exclude_patterns = ['_build']
-html_extra_path = ['./tmp']
```

Comment:
We need to include `tmp` in the path since we put the JVM and C API doc in it. See https://github.com/dmlc/xgboost/blob/f626799d251c20e183a8764a4c869514e3302c80/doc/conf.py#L36-L55

Reply:
I reverted the change, but the warning as error option is also removed as the tmp directory is not always available on PR branches due to httperror.